src/client/src/orangehrmLeavePlugin/pages/entitlements/EditEntitlement.vue

```diff
@@ -124,6 +124,7 @@ export default {
       window.appGlobal.baseUrl,
       '/api/v2/leave/leave-entitlements',
     );
+    http.setIgnorePath('/validation/entitlements');
```

Comment:
Added ignore path here because Invalid Parameter error was showing when entering a negative number for Entitlement

![image](https://user-images.githubusercontent.com/97493184/164369619-e95bb064-4d24-4399-acb9-e2354d204cef.png)

Reply:
Can't we go like this,
```suggestion
    http.setIgnorePath('/api/v2/leave/leave-entitlements/.+/validation/entitlements');
```
or
```js
http.setIgnorePath('/api/v2/leave/leave-entitlements/[0-9]+/validation/entitlements');
```